### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by emyrue**
- Used ES6 modules to write modular javascript
- Maintained functionality
- Used npm

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @emyrue  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, whose linters are not showing ingithub. Please add linters correctly and ask for another code review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
```
Comment (Amrhub) [inline modules/book.js]: - [ ] Kindly convert this to ES6 function (arrow function) as this is a practice to use ES6 syntax 😇

File: modules/menu.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export default function displaySection(menuList, menuAdd, menuContact,
```
Comment (Amrhub) [inline modules/menu.js]: - [ ] As mentioned above let's also make sure we convert this one.

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @emyrue,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emyrue** [comment]
I was unable to change the Book function to an arrow function because arrow functions do not have their own bindings to the "this" keyword.

**Review by gemmen29 — APPROVED** [review]
Hi @emyrue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**